### ISSUE TITLE
Suggested Improvement to Image Files Dialog preview image sizing element

### ISSUE THREAD

**Issue by hsankala** [cloudscribe/cloudscribe#623]
The image preview section of the `image files selection dialog` could be better sized to accommodate previewing of larger images.

![image](https://user-images.githubusercontent.com/14014749/56680710-aab0a200-66bf-11e9-963c-56fd0fb5d6f9.png)

At the moment the small preview window  is constrained to max-width:570px; height:390px; leaving you to use the scroll bars to peer at small sections of a large image 

I think it's hurting its usability when you want quick peak at the selected image. The mark up for the file preview is:
```
<div style="border:solid thin black; max-width:570px; height:390px;overflow:auto;margin-bottom:5px;">
                <img id="filePreview" src="/media/gallery-corner-cupboard/001-fitted-corner-cupboard.jpg" alt="Preview selected image">
            </div>
```

One solution would be to remove the fixed width and height style of the containing div element and apply the style 
    max-width: 100%;
    height: 400px;
to the filePreview img element this would enable you to see the whole image straight away.

**Comment by joeaudette** [comment]
I'll have to ponder this. I recall my intention being that I wanted it to be obvious if a user chooses a really large image. These days cameras have lots of mega pixels and image files that users upload can be huge. While we do have some resizing options to help with that users can also keep the original file. For example if you drop an image into the editor it does resize it but it links to the full size image. So if the user selects a full size image that may be several megabytes in size the scrollbars make that obvious whereas if we make any image fit in that view area without scrollbars users won't have any indication of the size and may use a full size image directly in content adding a huge payload to the page.

**Comment by CrispinF** [comment]
One solution here would be to allow the user to toggle between two views: 1:1 (as now) and "Fit" which does what Hefin is suggesting and shows the whole image.
Then if some basic metadata were also visible, like the dimensions and file size, all bases would be covered.

**Comment by joeaudette** [comment]
I've published updated nugets.

I've updated it to just set the height of the image so it auto adjusts the width to fit in the preview area. I've offset my previous concern by showing the file size in KB or MB as needed and if > 1MB I've added a warning about using the image in content. While it would be nice to show image dimensions that is an expensive process to get that information, it is not available from the FileInfo and would require loading the file in memory as an Image so not planning on doing that.
